test/docfx.Test/DocfxTest.cs

```diff
@@ -238,12 +304,74 @@ private static void RunBuild(string docsetPath, string outputPath, bool dryRun,
                 ? new Dictionary<string, string> { [".errors.log"] = errors }
                 : spec.Outputs;
 
-            VerifyOutput(randomOutputPath, outputs);
+            VerifyOutput(randomNormalOutputPath, outputs);
 
-            if (Directory.Exists(randomOutputPath))
+            if (isTwoStagesBuild)
             {
-                Directory.Delete(randomOutputPath, recursive: true);
+                CopyNecessaryFilesForContinue(randomNormalOutputPath, randomContinueOutputPath);
```

Comment:
Instead of copying files around, we only need to verify JSON files in 2 stage build